python/news.py

```diff
@@ -11,7 +12,7 @@
 
 
 class NewsService:
-    COMPANY_RELATION_URL = "https://openapi.companybooknetworking.com/1.0/news/company-relation"
+    COMPANY_RELATION_URL = "{}/{}".format(API_ENDPOINT_BASE, 'news/company-relation')
```

Comment:
Traditionally, double quotes would allow variable substitution while single quotes did not. Single quotes were faster as well. This is not the case anymore, substitution works in both cases and there is no speed difference. I don't think I actively decide on one or the other. Rather, it might be whatever I have done before. Maybe I should do a cleanup PR where a unify that.